cmpserver/plugin/plugin.go

```diff
@@ -203,6 +203,7 @@ func (s *Service) generateManifest(ctx context.Context, appDir string, envEntrie
 
 	manifests, err := kube.SplitYAMLToString([]byte(out))
 	if err != nil {
+		log.Debugf("Failed to split generated manifests. Beginning of generated manifests: %q", out[:1000])
```

Comment:
Can you do a length check? I think if `len(out) < 1000`, we'll get a runtime error.

Reply:
TIL